### PR TITLE
ostree branch tests from cli

### PR DESCRIPTION
**Pull request by sghai**

### CODE REVIEW

**Comment by sghai** [comment]
Referencing #3819

File: tests/foreman/cli/test_ostreebranch.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    @run_only_on('sat')
+    @tier1
+    def test_positive_list(self):
+        """List Ostree Branches
+
+        @id: 0f5e7e63-c0e3-43fc-8238-caf19a478a46
+
+        @Assert: Ostree Branch List is displayed
+        """
+        OstreeBranch.list()
```
Comment (oshtaier) [inline tests/foreman/cli/test_ostreebranch.py]: We need to add some validation here

**Comment by oshtaier** [comment]
Seems as really basic test cases :)

**Comment by oshtaier** [comment]
ACK

**Comment by oshtaier** [comment]
sorry

File: tests/foreman/cli/test_ostreebranch.py (hunk 1 above)
Comment (sghai) [inline tests/foreman/cli/test_ostreebranch.py]: sure

**Comment by sghai** [comment]
Yeah.. `ostree-branch` cmd  has only two sub-commands list and info. With this, I think we have good coverage around ostree/atomic under all cli/ui/api test modules.

**Comment by sghai** [comment]
test results:
```console
 robottelo]$ py.test -v tests/foreman/cli/test_ostreebranch.py::OstreeBranchTestCase
================================================================ test session starts =================================================================
platform linux2 -- Python 2.7.5, pytest-2.9.2, py-1.4.31, pluggy-0.3.1 -- /usr/bin/python
cachedir: .cache
rootdir: /home/sghai/QE/robottelo, inifile: 
plugins: cov-2.3.1, xdist-1.15.0, html-1.10.0
collected 6 items 

tests/foreman/cli/test_ostreebranch.py::OstreeBranchTestCase::test_positive_info_by_id <- robottelo/decorators/__init__.py 
PASSED
tests/foreman/cli/test_ostreebranch.py::OstreeBranchTestCase::test_positive_list <- robottelo/decorators/__init__.py PASSED
tests/foreman/cli/test_ostreebranch.py::OstreeBranchTestCase::test_positive_list_by_cv_id <- robottelo/decorators/__init__.py PASSED
tests/foreman/cli/test_ostreebranch.py::OstreeBranchTestCase::test_positive_list_by_org_id <- robottelo/decorators/__init__.py PASSED
tests/foreman/cli/test_ostreebranch.py::OstreeBranchTestCase::test_positive_list_by_product_id <- robottelo/decorators/__init__.py PASSED
tests/foreman/cli/test_ostreebranch.py::OstreeBranchTestCase::test_positive_list_by_repo_id <- robottelo/decorators/__init__.py PASSED

============================================================ 6 passed in 1289.73 seconds ===========================================================
```

**Comment by oshtaier** [comment]
ACK

**Comment by sghai** [comment]
One more ACK needed :)

**Pull request merged**